seed_reader/lib/routes/app_route.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import '../generated/l10n.dart';
+
+abstract class AppRoute {
+  bool canHandle(String route);
+  Route<T> handle<T>(RouteSettings settings);
+}
+
+class _RouteNotFound implements AppRoute {
+  const _RouteNotFound();
+
+  @override
+  bool canHandle(String route) => true;
```

Comment:
The RouteNotFound Route exists to cover the use case when someone is trying to push a named route that doesn't exist.
Since this route is the last route to be asked to handle, it should answer true.